packages/blade/src/components/Box/BaseBox/types/propsTypes.ts

```diff
@@ -99,9 +99,10 @@ type GridProps = MakeObjectResponsive<
 >;
 
 type ColorObjects = 'feedback' | 'surface' | 'interactive';
-type BackgroundColorString<T extends ColorObjects> = `${T}.background.${DotNotationToken<
-  Theme['colors'][T]['background']
->}`;
+type BackgroundOnlyColorObjects = 'popup' | 'overlay';
```

Comment:
We were only exposing 'feedback' | 'surface' | 'interactive' for background colors but we also need 'popup' | 'overlay'